modules/vstudio/vs2010_nuget.lua

```diff
@@ -56,283 +56,282 @@
 		-- that case we can examine the nuspec file and the file listing
 		-- locally.
```

Comment:
Maybe this comment should be moved below? (after the local functions definitions)